lib/livebook_web/live/hub/edit/team_component.ex

```diff
@@ -366,6 +369,13 @@ defmodule LivebookWeb.Hub.Edit.TeamComponent do
           </div>
         </div>
       </div>
+      <.link
+        :if={@confirm_url}
+        patch={@confirm_url}
+        class="button-base button-outlined-blue block text-center"
```

Comment:
This is a primary action, and I don't think we use a blue icon anywhere, so perhaps a primary button?

```suggestion
        class="button-base button-blue block text-center"
```